src/libpriv/rpmostree-postprocess.c

```diff
@@ -846,12 +846,22 @@ postprocess_selinux_policy_store_location (int rootfs_dfd,
 /* All "final" processing; things that are really required to use
  * rpm-ostree on the target host.
  */
-static gboolean
-postprocess_final (int            rootfs_dfd,
-                   JsonObject    *treefile,
-                   GCancellable  *cancellable,
-                   GError       **error)
+gboolean
+rpmostree_postprocess_final (int            rootfs_dfd,
+                             JsonObject    *treefile,
+                             GCancellable  *cancellable,
+                             GError       **error)
 {
```

Comment:
Let's do an auto error prefix here to keep the "Finalizing rootfs" ?

Reply:
Yes, nice catch!